python/lsst/jointcal/jointcal.py

```diff
@@ -40,23 +40,45 @@ class JointcalRunner(pipeBase.TaskRunner):
 
     @staticmethod
     def getTargetList(parsedCmd, **kwargs):
+        """
+        Return a list tuples per tract: (dataRefs, kwargs).
```

Comment:
"a list tuples"? Do you mean "a list of tuples"? I suggest something like: "Return a list of data refs and a kwargs dict" and then expand on what is in that kwargs dict below.